starknet_py/net/signer/stark_curve_signer.py

```diff
@@ -32,6 +34,18 @@ def from_private_key(key: Hash) -> "KeyPair":
             key = int(key, 0)
         return KeyPair(private_key=key, public_key=private_to_stark_key(key))
 
+    @staticmethod
+    def from_keystore(path: str, password: str) -> "KeyPair":
+        """
+        Create a key pair from a keystore file.
+        The keystore file should follow the Ethereum keystore format.
+        :param path: Path to the keystore file.
```

Comment:
A newline is necessary for correct formatting in the docs.
```suggestion
        The keystore file should follow the Ethereum keystore format.
        
        :param path: Path to the keystore file.
```